tests/test_breadth_first_combinations.py

```diff
@@ -0,0 +1,26 @@
+from typing import List, Tuple
+
+import pytest
+
+from tetra3 import breadth_first_combinations
+
+
+@pytest.mark.parametrize(
+    ("seq", "r", "expected"),
+    [
+        ([1, 2, 3], 1, [(1,), (2,), (3,)]),
+        ([1, 2, 3], 2, [(1, 2), (1, 3), (2, 3)]),
+        ([1, 2, 3], 3, [(1, 2, 3)]),
+        ([1, 2, 3, 4, 5, 6], 3, [(1, 2, 3), (1, 2, 4), (1, 3, 4), (2, 3, 4), (1, 2, 5), (1, 3, 5),
+                                 (2, 3, 5), (1, 4, 5), (2, 4, 5), (3, 4, 5), (1, 2, 6), (1, 3, 6),
+                                 (2, 3, 6), (1, 4, 6), (2, 4, 6), (3, 4, 6), (1, 5, 6), (2, 5, 6),
+                                 (3, 5, 6), (4, 5, 6)]),
+        ([1, 2, 3, 4, 5, 6], 4, [(1, 2, 3, 4), (1, 2, 3, 5), (1, 2, 4, 5), (1, 3, 4, 5),
```

Comment:
does this test case add any value coverage-wise or example-wise? I think we can drop it.

Reply:
removed